chaossearch/client/model.go

```diff
@@ -26,6 +26,7 @@ type ReadObjectGroupResponse struct {
 	PartitionBy      string
 	SourceBucket     string
 	IndexRetention   int
+	ArrayFlattenDepth *int
```

Comment:
can we use an int instead of a pointer here? The same way as we have it for everything else. I think it would be simpler

Reply:
We need a pointer to be able to set it to nil (unlimited flattening)